MetadataMigration/src/test/java/org/opensearch/migrations/EndToEndTest.java

```diff
@@ -0,0 +1,114 @@
+package org.opensearch.migrations;
+
+import java.io.File;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+import org.opensearch.migrations.metadata.tracing.MetadataMigrationTestContext;
+import org.opensearch.migrations.snapshot.creation.tracing.SnapshotTestContext;
+
+import com.rfs.common.ConnectionDetails.TargetArgs;
+import com.rfs.common.FileSystemSnapshotCreator;
+import com.rfs.common.OpenSearchClient;
+import com.rfs.framework.SearchClusterContainer;
+import com.rfs.http.ClusterOperations;
+import com.rfs.worker.SnapshotRunner;
+import lombok.SneakyThrows;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+/**
+ * Tests focused on setting up whole source clusters, performing a migration, and validation on the target cluster
+ */
+@Tag("longTest")
+class EndToEndTest {
+
+    @TempDir
+    private File localDirectory;
+
+    @Test
+    void metadataMigrateFrom_ES_v7_10_to_OS_v2_14() throws Exception {
+        final var targetVersion = SearchClusterContainer.OS_V2_14_0;
+        try (
+            final var sourceCluster = new SearchClusterContainer(SearchClusterContainer.ES_V7_10_2);
+            final var targetCluster = new SearchClusterContainer(targetVersion)
+        ) {
+            migrateFrom_ES_v7_X(sourceCluster, targetCluster);
```

Comment:
the expect/assert seems kind of hidden here -- we're expecting it to throw an Exception?

That's because we don't support 2.14? Can you add a little more clarification here on what/why. I assume long-term this won't throw an exception, and I want to make sure it's clear when we should change the expected behavior for this test.